README.md

```diff
@@ -1,2 +1,27 @@
-# fe-interview-design-system
-Public repository for the home test and pair programming exercise
+# Frontend Interview - Design System
+
+Hey 👋
+
+This is the base repository for the home test. The repository is created with `vite` and is empty, but contains some packages already installed, in particular:
+
+- `react`
+- `storybook`
+- `vitest`
+
+## Install and run
+
+```bash
+# Install dependencies
+# You can use also `npm` or `yarn`
+pnpm install
+
+# And run the project
+pnpm dev
+
+# Optional: Run Storybook
+pnpm storybook
+```
+
+## Figma file
+
+The figma file of the home test is available [here](https://www.figma.com/design/OclakAGLSXDoMKLFvwLNMP/%F0%9F%92%BB-Prima-Code-Challenge---Tabs-Component?node-id=0-1&t=4HapgpT4EAuIxz87-1), but you need the password to see it.
```

Comment:
I don't know if the person can ask question, maybe is better to make some more info from how he can retrieve the password like asking to xxx